src/main/java/com/spotinst/sdkjava/model/bl/azure/statefulNode/StatefulNodeGetStatusConfig.java

```diff
@@ -0,0 +1,285 @@
+package com.spotinst.sdkjava.model.bl.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by Bansi Chapla on 23/11/2021.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class StatefulNodeGetStatusConfig {
```

Comment:
Should it be named as StatefulNodeGetStatusResponse class

Reply:
Done